Libraries/LibGfx/ImmutableBitmap.cpp

```diff
@@ -31,6 +31,24 @@ int ImmutableBitmap::height() const
     return m_impl->sk_image->height();
 }
 
+int ImmutableBitmap::width_considering_exif() const
+{
+    if (!m_impl->source.has<NonnullRefPtr<Gfx::Bitmap>>())
+        return width();
+
+    auto const &bitmap = m_impl->source.get<NonnullRefPtr<Gfx::Bitmap>>();
+    return bitmap->width_considering_exif();
```

Comment:
You can simplify this a little by using `get_pointer` instead of `get`. It'll return a pointer to the value if it has one, and null if it doesn't, so it's equivalent to `has` and `get`, but only has to do the work once.